examples/sycl/02_bmg_gemm_mixed_dtype/02_bmg_gemm_f16_u4_f16.cpp

```diff
@@ -461,8 +461,8 @@ struct ExampleRunner {
     auto shape_B = cute::make_shape(N, K, L);
     auto shape_CD = cute::make_shape(M, N, L);
     auto shape_scale = cute::make_shape(dq_mn_size, scale_k, L);
-    auto shape_zero = [&]() {
-      if constexpr (is_tuple_v<std::remove_reference_t<decltype(cute::get<1>(stride_Z))>>) {
+    auto shape_zero = [&, stride_Z_ref = std::ref(stride_Z)]() {
```

Comment:
I prefer a solution without `std::ref`. Possible option: move the `is_tuple_v` out of the lambda and assign to variable.